pkg/logs/auditor/auditor.go

```diff
@@ -24,6 +24,12 @@ const defaultFlushPeriod = 1 * time.Second
 const defaultCleanupPeriod = 300 * time.Second
 const defaultTTL = 23 * time.Hour
 
+// FileOffsetStorage provides the last offset recorded by the agent after successfully sent data coming
+// from the file matching identifier
+type FileOffsetStorage interface {
```

Comment:
This is not ideal, the auditor shouldn't know about different sources. I'd rather slightly change the tailer

Reply:
Agreed, I changed this.